src/core/Memory.ts

```diff
@@ -1,9 +1,9 @@
-let values_in_browser: { date: Date, value: number }[] = [];
+export const storageKey = 'values';
```

Comment:
Ah, you export it for test.
As I mentioned, I don't think it is needed.

Reply:
i left it this way, cause there is one more usage in test